src/libPMacc/include/memory/buffers/MultiGridBuffer.hpp

```diff
@@ -220,7 +221,7 @@ class MultiGridBuffer
 
     GridBuffer<Type, DIM>& getGridBuffer(typename BufferNames::Names name)
     {
-        assert(name >= 0 && name < BufferNames::Count);
+        PMACC_ASSERT(name >= 0 && name < BufferNames::Count);
```

Comment:
You god me one second where I thought  this is a bug but than ...
The assert is only fire an exception if the expression is evaluated to `false`. This means that only valid names are allowed.